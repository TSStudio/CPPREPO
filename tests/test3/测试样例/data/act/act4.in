6
zhengqi 6
zitong 9
muhua 3
lili 4
xiaoh 5
zhangj 7