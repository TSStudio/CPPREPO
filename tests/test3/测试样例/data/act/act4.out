muhua
lili
xiaoh
zhengqi
zhangj
zitong